src/stories/Autocomplete.stories.tsx

```diff
@@ -49,21 +46,27 @@ Complex.args = {
   id: '2',
   label: 'Complex Objects',
   placeholder: 'Pick a value',
-  values: [{
-    label: 'hello',
-    value: 1,
-  }, {
-    label: 'world',
-    value: 2,
-  }, {
-    label: 'yoyo',
-    value: 3,
-  }, {
-    label: 'ma',
-    value: 4,
-  }],
+  values: [
+    {
+      label: 'hello',
+      value: 1,
+    },
+    {
+      label: 'world',
+      value: 2,
+    },
+    {
+      label: 'yoyo',
+      value: 3,
+    },
+    {
+      label: 'ma',
+      value: 4,
+    },
+  ],
   onChange: () => true,
   selected: undefined,
   hideClearIcon: true,
   errorText: '',
+  isEqual: (option: any, value) => option.label === value,
```

Comment:
@constanzauanini this is incorrect... the label is not the correct one to check. this will break things downstream.
The option.value is the correct one to check. if we aren't getting an object but just a string,,, this should be changed.